kfs-core/src/main/resources/edu/arizona/kfs/sys/businessobject/datadictionary/DocumentIncomeType.xml

```diff
@@ -3,22 +3,6 @@
 <!-- DV, PREQ and CM Document Income Type Business Objects -->
 <beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p" xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
 
-  <bean id="DocumentIncomeType" parent="DocumentIncomeType-parentBean"/>

```

Comment:
With some further looking, it appears this file was replaced with IncomeTypes.xml at some point in time. DocumentIncomeType.xml has been deleted.